x-pack/plugins/fleet/common/services/is_agent_upgradeable.test.ts

```diff
@@ -241,6 +242,154 @@ describe('Fleet - isAgentUpgradeable', () => {
   });
 });
 
+describe('Fleet - getNotUpgradeableMessage', () => {
+  it('if agent reports not upgradeable with agent version < latest agent version', () => {
+    expect(getNotUpgradeableMessage(getAgent({ version: '7.9.0' }), '8.0.0')).toBe(
+      'agent is marked as not upgradeable in elastic-agent.'
```

Comment:
```suggestion
      'agent cannot be upgraded through Fleet. It may be running in a container.'
```

cc @cmacknz - Any input on this particular message? I feel we don't want to surface something as technical as the supervisor process/PID 1 details here, but something explicit would be helpful until we get a more granular reason from https://github.com/elastic/kibana/issues/173281. Tried to borrow some wording from https://www.elastic.co/guide/en/fleet/current/elastic-agent-container.html#_what_you_need.

Reply:
My suggestion would be something like "It may be running in a container or is not installed as a service". 

That second scenario is the case where someone has just executed `./elastic-agent.exe` directly out of the installation directory.